conans/client/remote_manager.py

```diff
@@ -144,14 +144,12 @@ def get_package_info(self, package_reference, remote):
         returns (ConanInfo, remote_name)"""
         return self._call_remote(remote, "get_package_info", package_reference)
 
-    def get_recipe(self, conan_reference, dest_folder, remote):
+    def get_recipe(self, conan_reference, remote):
         """
         Read the conans from remotes
         Will iterate the remotes to find the conans unless remote was specified
 
         returns (dict relative_filepath:abs_path , remote_name)"""
-        rmdir(dest_folder)  # Remove first the destination folder
```

Comment:
Make sure the rmdir() executes to clean the folder before getting the recipe

Reply:
done